net.studio08.xbplay.yml

```diff
@@ -0,0 +1,63 @@
+app-id: net.studio08.xbplay
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: net.studio08.xbplay
+    buildsystem: simple
+    build-commands:
+      # setup build files
+      - cp -a . /app/main
+      - chmod +x /app/main/com.studio08.xbplay
+
+      # add icons
+      - install -Dm644 icon_32.png /app/share/icons/hicolor/32x32/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_64.png /app/share/icons/hicolor/64x64/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_128.png /app/share/icons/hicolor/128x128/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_256.png /app/share/icons/hicolor/256x256/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_512.png /app/share/icons/hicolor/512x512/apps/net.studio08.xbplay.png
+
+      # add desktop and metainfo files
+      - install -Dm644 net.studio08.xbplay.desktop -t /app/share/applications/
+      - install -Dm644 net.studio08.xbplay.metainfo.xml -t /app/share/metainfo/
+
+      # install the app runner
+      - install -Dm755 -t /app/bin/ run.sh
+    sources:
+      - type: archive
+        url: https://www.dropbox.com/s/333ffftl6qimhdq/data_1.1.0.zip?dl=1
+        sha256: cd4f1c59c5a920822460e99d423e69c7228c3ee92527dcc2b7500d46cf71da13
+        dest-filename: downloaded.zip
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/com.studio08.xbplay "$@"
+      - type: file
+        path: icon_32.png
+      - type: file
+        path: icon_64.png
+      - type: file
+        path: icon_128.png
+      - type: file
+        path: icon_256.png
```

Comment:
```suggestion
```

Reply:
Help me understand why I should remove these icons? Are they just not necessary?

Reply:
Yeah, 512x512 will be downscaled to 128x128 automatically.

Reply:
Understood, thanks for the details. I have removed all icons except for the 512 icon.